internal/build/docker.go

```diff
@@ -421,8 +422,11 @@ func readDockerOutput(ctx context.Context, reader io.Reader) (*json.RawMessage,
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
-		// TODO(Han): make me smarter! 🤓
-		logger.Get(ctx).Infof("%+v\n", message)
+		if len(message.Stream) > 0 && message.Stream != "\n" {
+			msg := strings.TrimSuffix(message.Stream, "\n")
+			logger.Get(ctx).Infof("%+v", msg)
+		}
+		// }
```

Comment:
👀

Reply:
<img width="1318" alt="output" src="https://user-images.githubusercontent.com/20349/44430001-633da900-a567-11e8-8cc7-e0a1231376af.png">

Reply:
Does this thumbs up constitute Approval? I'm itchy to merge.

Reply:
oh sorry @yuindustries, I saw that you merged branch in to master and my after 5 PM brain thought it said that you had already moved it in to master. Approving now!